intake_esm/aggregate.py

```diff
@@ -123,10 +157,12 @@ def concat_time_levels(dsets, time_coord_name_default, restore_non_dim_coords=Fa
             return dsets[0]
 
     attrs = dict_union(*[ds.attrs for ds in dsets])
+    time_coord_name = ensure_time_coord_name(dsets[0], time_coord_name_default)
 
+    # Equivalent to xr.align(*dsets, tolerance=xtol, exclude='time')
```

Comment:
This comment is not accurate. We are not testing the match between the coordinate values in `_override_coords`, we are simply dropping coordinates from dsets[1:] if the dimsizes match. Also, since the tolerance option doesn't even exist, this might be more confusing than helpful.

Reply:
Looking at it a bit more, the comment is very confusing :)  
I meant to add it as a TODO item once tolerance is supported in `xarray.align()`. I will just replace it with https://github.com/NCAR/intake-esm/issues/104#issuecomment-513404844 for future reference